app/controllers/items_controller.rb

```diff
@@ -96,17 +94,12 @@ def remove_collection
   end
 
   def register
-    @perm_keys = ["sunetid:#{current_user.login}"]
-    if webauth && webauth.privgroup.present?
-      @perm_keys += webauth.privgroup.split(/\|/).collect { |g| "workgroup:#{g}" }
-    end
-    render :register, :layout => 'application'
+    @perm_keys = current_user.groups
```

Comment:
Covered by existed tests. The old code is logically identical (copy/pasted) from the code under `groups`.